### ISSUE TITLE
it doesn't pull the text from the notes

### ISSUE THREAD

**Issue by osman1963** [michKneisel/obsidian-compile-notes-plugin#5]
Hi, 
Love the idea, and hope it's just a minor fix
It keeps the text I wrote in the original note and the links, just like making a copy from the note containing the links. It doesn't pull the text of the linked notes.
Creating the Manuscript folder was done correctly, and trashing the old Manuscript.md works also.
OS = Windows 10
Latest version of Obsidian

**Comment by michKneisel** [comment]
Hi, just check the settings for the notes folder and the location of your
linked notes. Its easier if you make the notes settings empty and your
linked notes all in the same folder as your source document.

**Comment by osman1963** [comment]
Got it. Now it works. Thanks a million.
BTW, any plans to make it with links to notes that are not in the same directory in the future, like search results for example?  Hope so ;)

**Comment by michKneisel** [comment]
I'm glad you got it working. It shouldn't be too difficult to add that functionality. My only concern was performance. But I would add it and toggle it on or off with a setting, so if it is a performance issue users can just turn it off. Thanks for the suggestion.

**Comment by michKneisel** [comment]
Ok, thank you! I will look into it.

**Comment by ksdavidc** [comment]
Yes, I see now. This only works if the linked files are in the same folder as the file that is being compiled. I am able to set the output folder correctly. 

Looking forwardto being able to select [[file#heading]] and [[file^blockid]] links
